### PR TITLE
[BLOG] add post on Python client

### PR DESCRIPTION
**Pull request by merobi-hub**
This change would add a post about the Python client to the blog.

### CODE REVIEW

File: contents/blog/python-client/index.mdx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+if parent_id:
+	parent_namespace, parent_job_name, parent_run_id = parent_id.split(‘/’)
+	parent_run_metadata = ParentRunMetadata(
+		run_id=parent_run_id,
+		job_name=parent_job_name,
+		job_namespace=parent_namespace
+	)
+
+processor = DbtArtifactProcessor(
+producer=PRODUCER,
```
Comment (rossturk) [inline contents/blog/python-client/index.mdx:79]: We have lost our indentation here.

File: contents/blog/python-client/index.mdx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+processor.dbt_run_metadata = dbt_run_metadata
+```
+
+After executing dbt, the script parses the metadata using the processor and emits a run event:
+
+```
+events = processor.parse().events()
+
+client.emit(dbt_run_event_end(
+run_id=dbt_run_metadata.run_id,
```
Comment (rossturk) [inline contents/blog/python-client/index.mdx:112]: I think we also need to indent this block for readability.

**Review by rossturk — CHANGES_REQUESTED** [review]
Looks great!! Two small indentation issues, then we are ready to merge.

File: contents/blog/python-client/index.mdx (hunk 2 above)
Comment (merobi-hub) [reply contents/blog/python-client/index.mdx]: thanks for catching!

**Review by rossturk — APPROVED** [review]

**Pull request merged**